presentation/common/src/presentation-common/PresentationManagerOptions.ts

```diff
@@ -211,14 +211,7 @@ export interface SingleElementPropertiesRequestOptions<TIModel, TParsedContent =
  * Request type for multiple elements properties requests.
  * @public
  */
-export interface MultiElementPropertiesRequestOptions<TIModel, TParsedContent = ElementProperties> extends RequestOptions<TIModel> {
-  /**
-   * Classes of the elements to get properties for. If [[elementClasses]] is `undefined`, all classes
-   * are used. Classes should be specified in one of these formats: "<schema name or alias>.<class_name>" or
-   * "<schema name or alias>:<class_name>".
-   */
-  elementClasses?: string[];
-
+export type MultiElementPropertiesRequestOptions<TIModel, TParsedContent = ElementProperties> = RequestOptions<TIModel> & {
   /**
```

Comment:
Do you or @ben-polinsky know what the docs for this complex type will look like on the docs site?

Reply:
Pulled from docs build artifact:
![image](https://github.com/user-attachments/assets/5dfb919c-c2b4-4d24-b55e-de703b27f455)

Reply:
Do the docs for the fields like `batchSize` show up?

Reply:
No. Restructured the types to make docs look better.

Reply:
Thanks. @ben-polinsky we might want to look into enhancements to the docs generator to avoid having to invent awkward intermediate types to get the docs properly formatted.

Reply:
Absolutely.